### ISSUE TITLE
Ruby >= 2.2.3 is actually required to build Vagrant, not just 2.0 like the docs say

### ISSUE THREAD

**Issue by gdubicki** [mitchellh/vagrant#7814]
In README.md and at https://www.vagrantup.com/docs/installation/source.html you say that "Ruby 2.0" is required, while when I actually tried to build it from current master head (2281c9) I get:

```
gdubicki@MacBook-Pro-Greg:~/git/vagrant (master)$ bundle install
Fetching https://github.com/mitchellh/vagrant-spec.git
(...)
Installing listen 3.1.5

Gem::InstallError: listen requires Ruby version >= 2.2.3, ~> 2.2.
(...)
An error occurred while installing listen (3.1.5), and Bundler cannot continue.
Make sure that `gem install listen -v '3.1.5'` succeeds before bundling.
```

Please update the docs then.

**Comment by gdubicki** [comment]
Sorry, I just read that the actual ruby version is in the `vagrant.gemspec` and it does in fact say that you need Ruby 2.2. But still it would be good to start saying ">= 2.2" in the docs.

**Comment by tas50** [comment]
Fixed by https://github.com/mitchellh/vagrant/pull/7819

**Comment by chrisroberts** [comment]
Closed via #7819
